endpoints/openrtb2/sample-requests/custom-currency/valid/custom-rates-override-usepbsrates-default.json

```diff
@@ -0,0 +1,68 @@
+{
+  "description": "Despite being more expensive, custom conversion rate overrides that of the currency conversion service. usepbsrates defaults to true",
+  "config": {
+    "currencyRates":{
+      "USD": {
+        "MXN": 2.00
+      }
+    },
+    "mockBidder": {
+      "currency": "USD",
+	  "price": 1.00
+    }
+  },
+  "mockBidRequest": {
+    "id": "some-request-id",
+    "site": {
+      "page": "test.somepage.com"
+    },
+    "imp": [
+      {
+        "id": "my-imp-id",
+        "video": {
+          "mimes": [
+            "video/mp4"
+          ]
+        },
+        "ext": {
+          "appnexus": {
+            "placementId": 12883451
+          }
+        }
+      }
+    ],
+    "cur": ["MXN"],
+    "ext": {
+      "prebid": {
+        "aliases": {
+          "unknown": "appnexus"
+        },
+        "currency": {
+          "rates": {
+            "USD": {
+              "MXN": 5.00
+            }
+          }
+        }
+      }
+    }
+  },
+  "expectedBidResponse": {
+      "id":"some-request-id",
+      "bidid":"test bid id",
+      "nbr":0,
+      "seatbid": [
+        {
+          "bid": [
```

Comment:
I did consider adding it to the `mockExchanges`'s implementation of `HoldAuction` but I changed my mind because at the end I wasn't super happy with these tests overall because I had to write a bit more features into the `mockExchange`  we use in `endpoints/openrtb2/auction_test.go`. I can go ahead and add the currency here but, what do you think? Do you agree with these `custom-currency/valid` tests? Or should I discard them along with the `mockExchange`'s added complexity. Strictly speaking, only the `custom-currency/errors` test cases are covering the error scenarios of the recently added `validateCustomRates()` function found in `endpoints/openrtb2/auction.go`. 

Let me know your take. I was enthusiastic about the `custom-currency/valid` folder at first but not so much now.